$OpenBSD$
Allow disabling Akonadi bindings building, until KDE bug 283677
gets fixed ( https://bugs.kde.org/show_bug.cgi?id=283677 ).
--- CMakeLists.txt.orig	Mon Oct 10 03:26:37 2011
+++ CMakeLists.txt	Mon Oct 10 03:28:17 2011
@@ -53,11 +53,13 @@ macro_optional_add_bindings(KDE4_FOUND "Plasma" plasma
 
 macro_optional_add_bindings(Soprano_FOUND "Soprano" soprano sopranoclient sopranoserver)
 
-if(KDEPIMLIBS_FOUND AND Akonadi_FOUND)
-    set(BUILD_Akonadi TRUE)
-else(KDEPIMLIBS_FOUND AND Akonadi_FOUND)
-    set(BUILD_Akonadi FALSE)
-endif(KDEPIMLIBS_FOUND AND Akonadi_FOUND)
+if(NOT DEFINED BUILD_Akonadi)
+    if(KDEPIMLIBS_FOUND AND Akonadi_FOUND)
+        set(BUILD_Akonadi TRUE)
+    else(KDEPIMLIBS_FOUND AND Akonadi_FOUND)
+        set(BUILD_Akonadi FALSE)
+    endif(KDEPIMLIBS_FOUND AND Akonadi_FOUND)
+endif(NOT DEFINED BUILD_Akonadi)
 macro_optional_add_bindings(BUILD_Akonadi "Akonadi" akonadi)
 
 macro_optional_add_bindings(LIBATTICA_FOUND "Attica" attica)
